### PR TITLE
doc: Update WRSI Zoom link

### PR DESCRIPTION
**Pull request by SMotaal**
This replaces the old zoom link with https://zoom.us/j/230136002

/cc @nodejs/website-redesign

### CODE REVIEW

**Comment by jonchurch** [comment]
I get a "Meeting ID is not valid" error when trying to join this link

**Comment by SMotaal** [comment]
> I get a "Meeting ID is not valid" error when trying to join this link

@jonchurch I get the same error :(

@keywordnew Once we have the permanent link, I will force commit from my end.

Any thoughts how to set it up?

**Comment by keywordnew** [comment]
Here is a new Zoom link: https://zoom.us/j/510926221

If you're able to join this link, it's good to be used for recurring meetings :-)

**Comment by joesepi** [comment]
@keywordnew is the link you posted above the one that will continue to be the regular zoom link? If so, can this PR be updated and merged? Thanks! /cc @SMotaal @jonchurch

**Comment by keywordnew** [comment]
The link in https://github.com/nodejs/create-node-meeting-artifacts/pull/84#issuecomment-586638101 doesn't seem to be valid anymore.

This is a new link for a recurring meeting that should not expire: Node.js Foundation is inviting you to a scheduled Zoom meeting.

https://zoom.us/j/671293205

File: templates/meeting_base_wrsi [hunk 1]
```diff
@@ -6,7 +6,7 @@ GROUP_NAME="Website Redesign Strategic Initiative"
 AGENDA_TAG=wr-agenda
 JOINING_INSTRUCTIONS="
 
-* Video conference link for participants: https://zoom.us/j/679487717
+* Video conference link for participants: https://zoom.us/j/230136002
```
Comment (keywordnew) [inline templates/meeting_base_wrsi]:
```suggestion
* Video conference link for participants: https://zoom.us/j/671293205
```

Comment (SMotaal) [reply templates/meeting_base_wrsi]: Do we commit this?

Comment (keywordnew) [reply templates/meeting_base_wrsi]: Yep. It's a recurring meeting that *shouldn't* expire. That was the issue happening previously.

**Comment by SMotaal** [comment]
LGTM

**Comment by keywordnew** [comment]
@mhdawson Please merge when you've got a moment, this PR has a new recurring Zoom to solve a persistent error on join :) Thanks!

**Pull request merged**